sdk/lib/cpprt/i386/cpprt.s

```diff
@@ -3,8 +3,8 @@
 .code
 
 MACRO(DEFINE_ALIAS, alias, orig)
-EXTERN &orig : PROC
-ALIAS <&alias> = <&orig>
+EXTERN < &orig > : PROC
+ALIAS < &alias > = < &orig >
```

Comment:
The documentation claims just one pair of brackets are needed, but here somehow you put these (OK) and down, where using the DEFINE_ALIAS, you put again angle brackets (so the macro should resolve with two angle brackets on the left and on the right 🤔 )
Also, why does it cause problems in this file, but not in the following ones ?? :
https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/vssapi/msvc.s;hb=91fceab36ee60c5380b5d48706974234b0d49100#l23
https://git.reactos.org/?p=reactos.git;a=blob;f=drivers/ksfilter/ks/msvc.S;hb=91fceab36ee60c5380b5d48706974234b0d49100#l10
https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/lib/cpprt/amd64/cpprt.s;hb=91fceab36ee60c5380b5d48706974234b0d49100#l5
https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/lib/crt/except/amd64/cpp.s;hb=91fceab36ee60c5380b5d48706974234b0d49100#l32
https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/lib/crt/except/i386/cpp.s;hb=91fceab36ee60c5380b5d48706974234b0d49100#l92

Reply:
Fixed. /Cp flag was missing.